applications/GeoMechanicsApplication/geo_mechanics_application_variables.cpp

```diff
@@ -243,4 +243,7 @@ KRATOS_CREATE_VARIABLE(double, STATE_VARIABLE_50)
 KRATOS_CREATE_VARIABLE(Vector, STRAINS_OF_PIECEWISE_LINEAR_LAW)
 KRATOS_CREATE_VARIABLE(Vector, STRESSES_OF_PIECEWISE_LINEAR_LAW)
 
+KRATOS_CREATE_VARIABLE(double, INTERFACE_NORMAL_STIFFNESS)
+KRATOS_CREATE_VARIABLE(double, INTERFACE_SHEAR_STIFFNESS)
```

Comment:
I see there's also a SHEAR_STIFFNESS. Wouldn't it be possible to add a NORMAL_STIFFNESS and re-use that instead of adding two new variables?

Reply:
We actually did consider what you suggested when we were developing this code. However, we wanted to be explicit about the context of these stiffness values. Therefore, we have added these new variables. The existing `SHEAR_STIFFNESS` variable is used in the context of continuum elements rather than interface elements.